crates/db/src/lib.rs

```diff
@@ -546,6 +638,133 @@ impl Database {
             .map_err(|err| DBError::DeactivateQuestFailed(Box::new(err)))
             .map(|_| id)
     }
+
+    async fn do_get_quest_reward_hook(
+        &self,
+        quest_id: &str,
+        tx: Option<&mut Transaction<'static, Postgres>>,
```

Comment:
Why this is `'static` when other functions use `'_` ?

Reply:
no reason